presto-hive/src/main/java/io/prestosql/plugin/hive/SyncPartitionMetadataProcedure.java

```diff
@@ -216,7 +216,8 @@ private static void dropPartitions(
                     session,
                     table.getDatabaseName(),
                     table.getTableName(),
-                    extractPartitionValues(name));
+                    buildPartitionObject(session, table, name),
+                    true);
```

Comment:
it actually should be `false`, as we think the partition does not exist

Reply:
it changes behaviour (it was true before) but makes sense. We don't want data here to be dropped :)